### ISSUE TITLE
[dev.icinga.com #321] change ndo* to ido* in the ido2db.cfg-sample and idomod.cfg-sample

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga-core#141]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/321

**Created by croft on 2010-03-03 11:33:24 +00:00**

Assignee: _mfriedrich_
Status: _Resolved_  (closed on _2010-05-21 10:25:05 +00:00_)
Target Version: _1.0.2_
Last Update: _2010-05-21 10:25:05 +00:00 (in Redmine)_

---

\#define NDOMOD\_PROCESS\_PROCESS\_DATA 1
\#define NDOMOD\_PROCESS\_TIMED\_EVENT\_DATA 2
\#define NDOMOD\_PROCESS\_LOG\_DATA 4
\#define NDOMOD\_PROCESS\_SYSTEM\_COMMAND\_DATA 8
\#define NDOMOD\_PROCESS\_EVENT\_HANDLER\_DATA 16
\#define NDOMOD\_PROCESS\_NOTIFICATION\_DATA 32
\#define NDOMOD\_PROCESS\_SERVICE\_CHECK\_DATA 64
\#define NDOMOD\_PROCESS\_HOST\_CHECK\_DATA 128
\#define NDOMOD\_PROCESS\_COMMENT\_DATA 256
\#define NDOMOD\_PROCESS\_DOWNTIME\_DATA 512
\#define NDOMOD\_PROCESS\_FLAPPING\_DATA 1024
\#define NDOMOD\_PROCESS\_PROGRAM\_STATUS\_DATA 2048
\#define NDOMOD\_PROCESS\_HOST\_STATUS\_DATA

Change all zu IDO...

**Changesets**

_2010-05-19 18:54:26 +00:00_ by _mfriedrich_ befc3c095ea58407cd86149844cc6721a61e81a5

    completely renamed idoutils (s/ndo/ido/g etc), removing old sources
    
    added icinga object version variable in neb api
    for usage with idomod. this requires both, icinga core
    and idoutils to be upgraded in 1.0.2
    
    basically, a full rename, dropping old references.
    
    fixes #231
    fixes #321

---

**Relations**:

* relates [#231](https://dev.icinga.com/issues/231)
